Assets/Scripts/Movements/WarpPointMovement.cs

```diff
@@ -50,7 +50,7 @@ public void Update()
 		}
 	}
 
-	private GameObject GetMainPlayerObject(GameObject obj)
+	public static GameObject GetMainPlayerObject(GameObject obj)

```

Comment:
I think this probably shouldn't be in WarpPointMovement. Maybe move it to SceneMessenger.